hypertrace-graphql-metric-schema/src/main/java/org/hypertrace/graphql/metric/request/DefaultMetricAggregationRequestBuilder.java

```diff
@@ -157,11 +167,20 @@ private Integer percentileSize(Map<String, Object> arguments) {
     AttributeModel attribute;
     AttributeModelMetricAggregationType aggregation;
     List<Object> arguments;
+    List<SelectedField> selectedFields;
```

Comment:
please don't put graphql framework constructs in the generated request - the whole purpose of the request builders is to parse the graphql objects into our own pojo. Instead, do this logic in the builder and set the appropriate value directly.